index.bs

```diff
@@ -204,14 +204,18 @@ NOTE: The dimensions indicated in the 'ispe' property might not match the values
 
 NOTE: The values of [=render_width_minus1=] and [=render_height_minus1=] possibly present in the AV1 bistream are not exposed in the AVIF container level.
 
+<h4 id="clean-aperture-property">Clean Aperture Property</h4>
+
+The semantics of the clean aperture property ('clap') as defined in [[!HEIF]] apply. In addition to the restrictions on transformative item property ordering specified in [[!MIAF]], the following restriction also applies:
+
+<assert>The origin of the 'clap' item property shall be anchored to 0,0 (top-left) unless the full, un-cropped image item is included as a secondary item in an image collection.</assert>
```

Comment:
Alternative: `essential` shall be equal to 0 for a 'clap' item property. Readers shall treat a `clap` item property as non-essential even if `essential` is 1.

Reply:
That would mean it's fine for a parser to always ignore the `clap`, even when the item is an input to some other derived item.

It would also mean it's fine for a parser to ignore a top-left anchored `clap`, which would lead to padding being shown for images that need the `clap` to deal with HW limitations.

Reply:
Right. I was thinking about implementations that parse but ignore the `clap` values and that do not expose the secondary non-hidden item.

As an example there is the libavif API that decodes to an `avifImage` which has a `clap` field. That `clap` field is ignored by `avifdec` which always outputs a PNG containing the full, uncropped image. I guess this is not compliant but I can imagine others doing the same, just accessing the pixels and ignoring the multiple obscure metadata chunks.

So I would have liked to adapt the spec to just accept the current state of things, at least for items that do not have further transformations, but that is beyond this PR I guess.

Reply:
Ah. I see your point.

I think the real fix would be for `avifdec` to apply the crop when writing to PNG though. From what I understand from the conversation where Chrome resurrected support for `clap`, there was a real use-case where it was needed for some HW that had size limitations. The current behaviour from `avifdec` will decode and show the padding for those files which I don't think is the correct behaviour.

Reply:
Is Chrome applying `clap` now? https://github.com/AOMediaCodec/libavif/issues/1177 can be closed then.

Reply:
I filed https://github.com/AOMediaCodec/libavif/issues/2427.

Reply:
> Is Chrome applying `clap` now? [AOMediaCodec/libavif#1177](https://github.com/AOMediaCodec/libavif/issues/1177) can be closed then.

I think so at least:
https://github.com/chromium/chromium/commit/3a13337543c1e7de6914f87cd6f02ab06751c572

Reply:
https://issues.chromium.org/issues/40054481